rest/model/admin.go

```diff
@@ -908,7 +910,8 @@ func (a *APIPodInitConfig) BuildFromService(h interface{}) error {
 
 func (a *APIPodInitConfig) ToService() (interface{}, error) {
 	return evergreen.PodInitConfig{
-		S3BaseURL: utility.FromStringPtr(a.S3BaseURL),
+		S3BaseURL:              utility.FromStringPtr(a.S3BaseURL),
+		MaxParallelPodRequests: a.MaxParallelPodRequests,
```

Comment:
I would add an assertion for this extra field in the admin REST model test like [this one](https://github.com/evergreen-ci/evergreen/blob/52a73f7fee9d3f8cf56cf52b96e811b213cc955a/rest/model/admin_test.go#L136) for `BuildFromService` and [this one](https://github.com/evergreen-ci/evergreen/blob/52a73f7fee9d3f8cf56cf52b96e811b213cc955a/rest/model/admin_test.go#L218) for `ToService`.

I would also add an assertion for [the REST route test](https://github.com/evergreen-ci/evergreen/blob/52a73f7fee9d3f8cf56cf52b96e811b213cc955a/rest/route/admin_test.go#L150).

Reply:
Still missing the assertion for the REST route test.